packages/airnode-deployer/test/mock-utils.ts

```diff
@@ -0,0 +1,14 @@
+// NOTE: This file is referenced as a pathGroup pattern in .eslintrc (import/order)
+
+/**
+ * Mocks the console for silenced tests and otherwise enables it for local development.
+ */
+export const mockConsole = () => {
+  if (process.env.SILENCE_LOGGER) {
+    // Mock console log for github
```

Comment:
Github shouldn't behave differently to local environment. Are you sure you didn't have some wierd setup locally?

Reply:
Yes definitely I can look into this more